### PR TITLE
feat(InterceptElytra): intercept elytra flyers using wind charges (#8718)

### LINKED ISSUE #8718: [FEATURE] Add Intercept Elytra module (Wind Charge anti-elytra counter)
### Describe your feature request.

I'd like to request a new module: Intercept Elytra.
This module is designed to counter players using Elytra without fireworks - a common situation in Mace PvP.

How it works:

1. Detects an incoming player gliding with Elytra (no firework boost, just gliding).
2. Calculates the trajectory of the gliding player.
3. Automatically throws a Wind Charge with high precision to intercept the target.
4. The Wind Charge knocks the player back and disrupts their flight trajectory, preventing them from landing a mace critical hit on you.

### Additional context

Wind Charges are known for their strong knockback effect, making them ideal for disrupting Elytra flight .
The Breeze's wind charge attack works similarly — it knocks players back and disrupts movement .
This module is the perfect counterpart to AutoPearlCatch, completing the Mace PvP toolkit: one for your own mobility, one for stopping enemies.

### PR DESCRIPTION
**Pull request by CompileRider**
## Description

Implements #8718 — **InterceptElytra**: a new Combat module that intercepts elytra flyers by launching Wind Charges at their predicted position to knock them out of the air.

### How it works
- **Target selection** — O(k) single-pass scan (no sorting, O(1) auxiliary space) picks the closest gliding player within range, with lazy line-of-sight evaluation.
- **Closed-form interception solver** — solves the quadratic `(|w|² − 2.25)t² + 2(Δ·w)t + |Δ|² = 0` in **O(1)** (wind charge speed = 1.5 blocks/tick, inherits the thrower's velocity), with vanilla-accurate knockback prediction (`ServerExplosion` formula).
- **Two aim modes** — `InterceptPoint` (leads the target) and `Direct` (fires at the current position).
- **VerifyHit** — simulates the wind-charge trajectory and validates it against the predicted impact point before committing to a shot.
- **15 settings** — range, minimum target speed, require-gliding toggle, vertical offset, throw mode, etc.

### Tests
26 unit tests across 3 classes (`InterceptElytraSolverTest`, `ElytraKnockbackCalculatorTest`, `GliderPredictionTest`), including a round-trip check against continuous vanilla flight, NaN/Inf guards and escaping-target edge cases. `./gradlew build` (compile + test + detekt) passes.

Closes #8718

### CODE REVIEW

**Comment by CompileRider** [comment]
Draft: I will test this in-game first and will mark it ready for review only once the behavior is verified in a live session.

**Comment by CompileRider** [comment]
Note: the branch was rebased onto the latest `nextgen` and now contains only the InterceptElytra changes (18 files) — the unrelated config-serialization refactor mentioned above is no longer part of this PR.

**Comment by CompileRider** [comment]
I am currently traveling.

**Comment by CompileRider** [comment]
I tested this ingame and it works as described: it locks onto a gliding player, leads them with the wind charge and knocks them out of the air. The unit tests and detekt still pass.

Marking it ready for review.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/interceptelytra/ModuleInterceptElytra.kt [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    /** Casts to a gliding player that can be knocked out of the air, or null when not targetable. */
+    private fun asTargetableGlider(entity: Entity): LivingEntity? {
+        val glider = entity as? Player ?: return null
+        val hasSpeed = glider.deltaMovement.horizontalDistance() >= minimumTargetSpeed
+
+        return glider.takeIf { candidate ->
+            candidate !== player &&
+                candidate.isAlive &&
+                (!requireGliding || candidate.isFallFlying) &&
+                !candidate.abilities.flying && // immune to wind charge knockback
```
Comment (MukjepScarlet) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/interceptelytra/ModuleInterceptElytra.kt]: `candidate.abilities.instabuild` (See `ModuleAutoLeave`)

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/interceptelytra/ModuleInterceptElytra.kt [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+            useHotbarSlotOrOffhand(slot, slotResetDelay.random(), aim.rotation.yaw, aim.rotation.pitch)
+            chronometer.reset()
+        }
+    }
+
+    /**
+     * Selects the closest valid glider in a single O(k) pass without sorting or temporary
+     * collections (O(1) auxiliary space).
+     */
+    private fun selectBestGlider(): LivingEntity? {
```
Comment (MukjepScarlet) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/interceptelytra/ModuleInterceptElytra.kt:118]: No teammate check?

Comment (CompileRider) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/interceptelytra/ModuleInterceptElytra.kt:118]: I hadn't thought of that

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/interceptelytra/ModuleInterceptElytra.kt [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    private val verifyHit by boolean("VerifyHit", false)
+    private val rotations = tree(RotationsValueGroup(this))
+
+    private val chronometer = Chronometer()
+
+    private val cooldownReached: Boolean
+        get() = chronometer.hasElapsed((cooldown.random() * MILLISECONDS_PER_TICK).toLong())
+
+    @Suppress("unused")
+    private val interceptHandler = tickHandler {
```
Comment (MukjepScarlet) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/interceptelytra/ModuleInterceptElytra.kt:88]: Please use `handler` for non-suspend callback.

File: src/main/resources/resources/liquidbounce/lang/de_de.json [hunk 4]
```diff
@@ -305,6 +305,7 @@
   "liquidbounce.module.disabler.messages.grimSpectateEnableMessage": "Aktivier diesen Disabler beim Respawnen – wenn du Fliegen kannst. Du solltest in der Lage sein, herumzufliegen und Blöcke zu zerstören, dabei aber keine anderen Spieler angreifen können. Deaktivier den Disabler es so schnell wie möglich, sonst kommt es zu einer Zeitüberschreitung.",
   "liquidbounce.module.eagle.description": "Schleicht automatisch am Rand eines Blocks.",
   "liquidbounce.module.elytraFly.description": "Erleichtert die Steuerung der Elytra.",
+  "liquidbounce.module.interceptElytra.description": "Wirft Windladungen auf Elytra-Gleiter, um sie abzufangen und aus der Luft zu stoßen.",
```
Comment (MukjepScarlet) [inline src/main/resources/resources/liquidbounce/lang/de_de.json:308]: Please update i18n files again because of main branch sorted the files.
